src/components/ContactSelector/ContactSelector.js

```
@@ -83,33 +173,132 @@ const ContactSelector = ({
+        <Icon name="down-arrow" color={colors.text} width={10} height={10} style={{ marginLeft: 6 }} />
       </SelectedOption>
     );
   };
 
+  if (!selectedContact && !isKeyBasedAccount) {
+    return (
+      <SelectionContainer>
+        {selectorList?.map((item) => {
+          if (!item) return null;
+          return (
+            <ButtonContainer onPress={item.onPress}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)